bin/initoverlayfs-install

```diff
@@ -37,6 +37,9 @@ exec_erofs() {
     popd
     rm -f "${INITRAMFS_DIR}/initoverlayfs-$kver.img"
     mkfs.erofs $erofs_compression "${INITRAMFS_DIR}/initoverlayfs-$kver.img" ${INITRAMFS_DUMP_DIR}
+    if false; then
```

Comment:
Hey Martin, I believe the if here is not needed as we have the `set -ex` in the top of the script but I am not sure, could you please double check?

Reply:
We should leave this in for now, it's gonna eventually become an "if ostree" kind of statement.

We don't want to do dm-verity for regular images, it has a performance cost and we don't have a secure boot story for non-ostree images.

Reply:
ahh okay, thanks for the clarification.

```
OSTREE_BIN="/usr/bin/ostree"

if [ -e "$OSTREE_BIN" ]; then
       foobar
fi
```

Reply:
We can leave if false in for now just to make progress and not block, I have ideas for the if OSTree check once the rpm-ostree integration is a bit futher along and we can boot an unverified OSTree image with initoverlayfs

Reply:
This is one of many changes we will need in initoverlayfs-install to accommodate for the rpm-ostree flow...